morpheus/stages/inference/triton_inference_stage.py

```diff
@@ -405,9 +405,9 @@ def build_input(self, name: str, data: cp.ndarray, force_convert_inputs: bool) -
 
 
 # This class is exclusively run in the worker thread. Separating the classes helps keeps the threads separate
-class _TritonInferenceWorker(InferenceWorker):
+class TritonInferenceWorker(InferenceWorker):
```

Comment:
My thoughts were that the previous workers such as `TritonInferenceNLP` were public, then the base class they were folded into should also be public.

If the log parsing example needs to subclass `TritonInferenceWorker`, then yes, it should be public. 

If on the other-hand it can work without subclassing, then I agree it should remain private.

Reply:
I think it should be public, the primary reason is if a user wants a custom Triton inference worker, currently the only public class is the `InferenceWorker` requiring the user to replicate code currently in the `_TritonInferenceWorker` class.

With that said, the counter-argument is that I think the primary reason a user would need a custom worker would be to handle tensors in addition to the `probs` tensor. Which could be a constructor argument, which I believe would allow us to remove the custom stage from the log parsing example.